elastic-search-support/src/main/resources/globalSearchCasesMapping.json

```diff
@@ -0,0 +1,195 @@
+      "search_analyzer": "alphaNumericStringAnalyzer"
+    },
+    "caseNameHmctsInternal": {
+      "type": "text",
+      "fields": {
+        "keyword": {
+          "type": "keyword",
+          "ignore_above": 256,
+          "normalizer": "lowercase_normalizer"
+        }
+      },
+      "analyzer": "standard",
+      "search_analyzer": "alphaNumericStringAnalyzer"
+    },
+    "caseManagementCategoryName": {
+      "type": "text",
+      "fields": {
+        "keyword": {
+          "type": "keyword",
+          "ignore_above": 256,
+          "normalizer": "lowercase_normalizer"
+        }
+      },
+      "analyzer": "standard",
+      "search_analyzer": "alphaNumericStringAnalyzer"
+    },
+    "OtherCaseReferences": {
+      "properties": {
+        "id": {
+          "enabled": false
+        },
+        "value": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword",
+              "ignore_above": 256,
+              "normalizer": "lowercase_normalizer"
+            }
+          },
+          "analyzer": "standard",
+          "search_analyzer": "alphaNumericStringAnalyzer"
+        }
+      }
+    },
+    "SearchParties": {
+      "properties": {
+        "id": {
+          "enabled": false
+        },
+        "value": {
+          "properties": {
+            "Name": {
+              "type": "text",
+              "fields": {
+                "keyword": {
+                  "type": "keyword",
+                  "ignore_above": 256,
+                  "normalizer": "lowercase_normalizer"
+                }
+              },
+              "analyzer": "standard",
+              "search_analyzer": "alphaNumericStringAnalyzer"
+            },
+            "EmailAddress": {
+              "type": "text",
+              "fields": {
+                "keyword": {
+                  "type": "keyword",
+                  "ignore_above": 256,
+                  "normalizer": "lowercase_normalizer"
+                }
+              },
+              "analyzer": "standard",
+              "search_analyzer": "alphaNumericStringAnalyzer"
+            },
+            "AddressLine1": {
+              "type": "text",
+              "fields": {
+                "keyword": {
+                  "type": "keyword",
+                  "ignore_above": 256,
+                  "normalizer": "lowercase_normalizer"
+                }
+              },
+              "analyzer": "standard",
+              "search_analyzer": "alphaNumericStringAnalyzer"
+            },
+            "PostCode": {
+              "type": "text",
+              "fields": {
+                "keyword": {
+                  "type": "keyword",
+                  "ignore_above": 256,
+                  "normalizer": "lowercase_normalizer"
+                }
+              },
+              "analyzer": "standard",
+              "search_analyzer": "alphaNumericStringAnalyzer"
+            },
+            "DateOfBirth": {
```

Comment:
We could use the existing configuration for dates so Elasticsearch treats them properly and allows date-related queries - see [here](https://github.com/hmcts/ccd-definition-store-api/blob/master/elastic-search-support/src/main/resources/application.yml#L15)

Reply:
done